docs/reference/slate-react/plugins.md

```diff
@@ -109,7 +109,15 @@ This handler is called whenever the native DOM selection changes.
 
 `Function renderEditor(props: Object, next: Function) => ReactNode|Void`
 
-The `renderEditor` property allows you to define higher-order-component-like behavior. It is passed all of the properties of the editor, including `props.children`. You can then choose to wrap the existing `children` in any custom elements or proxy the properties however you choose. This can be useful for rendering toolbars, styling the editor, rendering validation, etc. Remember that the `renderEditor` function has to render `props.children` for editor's content to render.
+The `renderEditor` property allows you to define higher-order-component-like behavior. It is passed all of the properties of the editor, including `children`, which you can access as `next()`. You can then choose to wrap the existing `children` in any custom elements or proxy the properties however you choose. This can be useful for rendering toolbars, styling the editor, rendering validation, etc. Remember that the `renderEditor` function has to render `children` for editor's content to render. For example:
+```js
+renderEditor: ({ editor }, next) => (
+  <div>
+    <MyToolbarComponent editor={editor} />
+    <MyEditorComponent editor={editor}>{next()}</MyEditorComponent>
```

Comment:
Can you set this as a variable `const children = next()` to make it more explicit for people?